js/index.js

```diff
@@ -1 +1,84 @@
 // Your code goes here
+var btn = document.querySelectorAll('.btn');
+var modal = document.querySelector(".modal");
+var modalText = document.querySelector('.modal-content p')
+
+
+
+btn.forEach(btn => btn.addEventListener('mousedown', event => {
+    modal.style.display = 'block';
+   
+}));
+
+var span = document.getElementsByClassName("close")[0];
```

Comment:
for readability, I would call this variable closeBtn or something similar  it gets a bit tricky to follow the code if the variables aren't clearly identified.